src/parser/transform/transform_expression.cpp

```diff
@@ -312,15 +312,15 @@ std::unique_ptr<ParsedExpression> Transformer::transformListOperatorExpression(
             left = transformExpression(*ctx.oC_Expression(0));
             right = transformExpression(*ctx.oC_Expression(1));
         } else if (ctx.oC_Expression().size() == 0) { // [:]
-            left = std::make_unique<ParsedLiteralExpression>(Value(0), "0");
-            right = std::make_unique<ParsedLiteralExpression>(Value(0), "0");
+            left = std::make_unique<ParsedLiteralExpression>(Value(1), "1");
```

Comment:
I thought we only change the logic for handling right value. Do left value needs to change as well?